### ISSUE TITLE
query CK crowdnode status from CK machine module

### ISSUE THREAD

**Issue by gfursin** [ctuning/ck-crowdnode#12]
We should obtain various parameters automatically (including full path, etc) when registering CK crowdnode in the CK machine module (to be more user-friendly) using new status function. It is not critical though at this moment.

**Comment by DVEfremov** [comment]
I've already implemented status command at commit dd94fd4514cf2bde0e544cca55b03d5e6628bc3a
you can try it with command:
`ck status remote-ck-node:: --remote_server_url=http://localhost:3333 --secretkey=<secretkey>`
and get result like this: 
`{"return":"0","path_to_files":"/home/daniil/ck-crowdnode-files"}`
Is that you've expected ?

**Comment by gfursin** [comment]
Yes - thanks! Is it just possible to add "cfg" key there, please, i.e.:
{"return":"0","cfg":{"path_to_files":"/home/daniil/ck-crowdnode-files"}}
Since we may add more keys or other information groups ...

This ticket is about adding some extra functionality in ''machine'' module to query CK crowdnode using ''status'' and then fill in required info. It shouldn't be very difficult to do and I may actually do it at some point - just have a few other urgent things to do right now ;) ...

**Comment by DVEfremov** [comment]
no prob
